protocol.md

```diff
@@ -373,16 +373,16 @@ Prerequisite: the *user*'s *device* is authenticated against the *Tanker server*
 8. *Tanker Core* encrypts the private [Group Encryption Key Pair] with the *application* [Public Provisional Identity]'s encryption key
 9. *Tanker Core* encrypts the result of the previous step with the *Tanker* [Public Provisional Identity]'s encryption key
 10. Using the private [Group Signature Key Pair], *Tanker Core* signs the public and encrypted private [Group Encryption Key Pair] and [Group Signature Key Pair]
-11. *Tanker Core* creates a `user_group_creation` *block* with all of the above and pushes it to the *Trustchain*
+11. *Tanker Core* creates a `user_group_creation` *block* with all the above and pushes it to the *Trustchain*
 12. The *Tanker server* validates and holds the block until it is claimed
 
 ### Claiming a provisional identity
 
 Prerequisite: the *user*'s *device* is authenticated against the *Tanker server* and some *users* have shared *data* with a provisional identity owned by them or added it to a group.
 
-1. The *application server* sends an email to the *user* with a verification code
+1. The *application server* sends a verification code to the *user* (via email or SMS, according to their verification method)
```

Comment:
according to their verification method -> depending on the verification method associated with the provisional identity

so that the reader doesn't not confuse it with the user's verification method

Reply:
I used your wording, though I'm unsure that we don't _want_ it to be confused with the user's verification method, since attaching a provisional does use the user's verification method to avoid forcing a second verification?